### PR TITLE
Was missing the location of the master files that many people ask for in the overview

### PR DESCRIPTION
**Pull request by BrittanyBunk**
Added this in: 
After reading the tech and developments for a background on Infogami, the most recent files can be viewed and worked on.

- [Infogami master files](https://github.com/internetarchive/infogami)

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -58,6 +58,10 @@ Once you've read the overview of OpenLibrary Backend technologies, it's highly e
 
 - [Infogami Developer Tutorial](https://openlibrary.org/dev/docs/infogami)
 
+After reading the tech and developments for a background on Infogami, the most recent files can be viewed and worked on.
+
+- [Infogami master files](https://github.com/internetarchive/infogami)
```
Comment (cclauss) [inline Readme.md:63]:
```suggestion
After reading the Infogami technology overview and developer tutorial, please visit the [Infogami repository](https://github.com/internetarchive/infogami) to see if there are places where you can contribute.
```

**Comment by cdrini** [comment]
I'm leaning towards closing this one; Infogami isn't really being actively maintained except for the python 3 migration. Do you have an example of users who were looking for these files?

**Comment by cclauss** [comment]
I believe that we should land this one.  This is a _First-time contribution_ and we should take care to be extra welcoming on those.  In the past week I have recommended that two potential contributors take a look at Infogami so I see no reason to hide it while it is still our dependency and it still needs to be upgraded.  Do we have immediate plans to remove Infogami without improving it?

**Comment by cdrini** [comment]
Oh that's my bad! We've communicated with @BrittanyBunk so often over slack I didn't even realize this was her first contribution :) I understand those concerns, but this is also the main README for the project, so keeping it informative for future first-timers is also important. The scenario I'm afraid of is a contributor reading this and and then going to the infogami repo and expecting a community there. Infogami is a bit of a ghost town, and we just don't have the bandwidth to change that. But let me see if I can find a way to make the markup meet both of these needs...

File: Readme.md (hunk 1 above)
Comment (cdrini) [inline Readme.md:63]:
```suggestion
If you want to dive into the source code for Infogami, see the [Infogami repo](https://github.com/internetarchive/infogami).
```

- Removed the bullet point to reduce visual weight; this is less essential for OL development than the other 2 points.
- Used `If ...` to clarify this is an optional, not essential step of getting familiar with Open Library's backend.

How's this @cclauss @BrittanyBunk ?

Comment (BrittanyBunk) [reply Readme.md:63]: Perfect!

**Comment by BrittanyBunk** [comment]
> Do you have an example of users who were looking for these files?

Many newcomers who want to delve into code ask me where they can find the Infogami files. 

@cdrini I didn't know that Python 3 would replace Infogami. I just thought it was an upgrade from Python 2 as I heard it's discontinued. If what you're saying is true though, I would gut out all the other references to using Infogami after the transition and just keep: 'In the past we used Infogami. (about | source code). Now we use Python 3 (about | source code) and Solr (about | source code)'

**Comment by cclauss** [comment]
To move OpenLibrary to Python 3, we have 3 choices:
1. Port Infogami so it runs on both Python 2 and Python 3
2. Run Infogami on Python 2 and OpenLibrary on Python 3
3. Replace Infogami with something else

**Comment by BrittanyBunk** [comment]
@cclauss If that is true, then what is the issue that @cdrini is talking about then?

**Comment by cdrini** [comment]
@BrittanyBunk Sorry, Python 3 won't replace infogami; the python 2 to python 3 migration is currently the only change we have planned for infogami. Infogami will likely still be around (although I don't know how I feel about that 😅 ).

**Comment by BrittanyBunk** [comment]
@cdrini thank you. Yes, that's what I thought. So I would say we still will need this as long as Infogami's used and people request to see it (as it's their starting point when they visit the site).

**Pull request merged**